examples/light-js/index.html

```diff
@@ -87,6 +108,37 @@
 
       statusDiv.innerHTML = "<p>Starting Waku node.</p>";
       await node.start();
+
+      // Queries all peers from store and updates list of connected peers
+      const updatePeersList = async () => {
+        // Generate <p> element with connection string from each peer
+        const peers = await node.store.peers();
```

Comment:
`node.store` refers to `Store` protocol and though it seems to be possible to get peers from there we should try using `connectionManager` or `libp2p` directly.

![image](https://github.com/waku-org/js-waku-examples/assets/118575614/621c1a3a-4dbe-4d60-a73a-166486bbd2c5)

Reply:
Updated to use `node.libp2p.peerStore`